lib/babel-istanbul-instrumenter.js

```diff
@@ -0,0 +1,349 @@
+/* global Symbol */
```

Comment:
This was from `isparta` which is not maintained anymore.  I actually meant to clean up my fork so it doesn't look like it came straight out of `babel`.